doc/guidance.md

```diff
@@ -14,9 +14,22 @@ This is about how to use Doxygen to maintain the API documents. There are three
 	- on mac, run "brew install graphviz"
 - goto the root directory of "amazon-freertos" (this will be referenced as $ROOT_AFR_DIR)
 	- cd $ROOT_AFR_DIR
-- run Doxygen command to generate the documents 
-	- doxygen doc/config/main
+- create a tag folder to store doxygen generated tags
+	- mkdir $ROOT_AFR_DIR/doc/tag
```

Comment:
Would it be better to just check in a tag folder?

Reply:
What would be better is to change the tags to go to the output folder instead. Git does not allow empty folders, so just checking in an empty tag folder is not possible, you would have to place a dummy file which is messy.

I do not see the value of a separate tag folder, I would rather just place the tag files in the output folder and then it would all work without any issues always.

Reply:
Done